### DISCUSSION TITLE
HDMI Trigger

### DISCUSSION THREAD

**Discussion by FPVMartin** [gopro/labs#678] [Q&A] [answered]
Hi,

Is there a way to have the GoPro power on (wake) when HDMI is plugged to the GoPro Media Mod?

Thanks!

Martin

**Comment by dnewman-gpsw** [comment, marked as answer]
Not currently, and I expect it wouldn't be possible.  The USB trigger works, as power is added (camera likes power).  HDMI would not be powering the camera, so it will not enable the camera to boot.  If you also add USB power to the Media Mod, you can use that as a boot trigger.

**Reply by FPVMartin** [reply]
Ok thanks. I see that the GoPro detects HDMI when it’s plugged in. I thought it could be used as a trigger.
